infrastructure-scripts/codebase_loc.sh

```
@@ -0,0 +1 @@
+find . -type f \( -name "*.py" -o -name "*.js" -o -name "*.html" \) -exec wc -l {} + | awk '{total += $1} END {print total}'
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)